gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -77,6 +78,10 @@ public class AndroidCommandPluginExtension {
         System.properties['seed'] ?: seed
     }
 
+    def getCategories() {
+        System.properties['categories'] ?: categories ?: ''
```

Comment:
wait a sec, are the `System.properties['categories']` not the same as `categories`? 
could this be `System.properties['categories'] ?: ' '`?